packages/ui/components/input-file/src/LionInputFile.js

```diff
@@ -0,0 +1,848 @@
+import { LionField } from '@lion/ui/form-core.js';
+import { LocalizeMixin } from '@lion/ui/localize.js';
+import { ScopedElementsMixin } from '@open-wc/scoped-elements';
+import { css, html } from 'lit';
+import { FileHandle, MAX_FILE_SIZE } from './FileHandle.js';
+import { LionUploadedFileList } from './LionUploadedFileList.js';
+import { localizeNamespaceLoader } from './localizeNamespaceLoader.js';
+import { DuplicateFileNames, FileValidation } from './validators.js';
+
+/**
+ * @typedef {import('lit').TemplateResult} TemplateResult
+ * @typedef {import('lit').RenderOptions} RenderOptions
+ * @typedef {import('../types/input-file.js').InputFile} InputFile
+ * @typedef {import('../types/input-file.js').SystemFile} SystemFile
+ * @typedef {import('../types/input-file.js').UploadResponse} UploadResponse
+ */
+
+/**
+ * @param {number} bytes
+ * @param {number} decimals
+ */
+function formatBytes(bytes, decimals = 2) {
+  if (!+bytes) {
+    return '0 Bytes';
+  }
+  const k = 1024;
+  const dm = decimals < 0 ? 0 : decimals;
+  const sizes = [' bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+  const i = Math.floor(Math.log(bytes) / Math.log(k));
+  return `${parseFloat((bytes / k ** i).toFixed(dm))}${sizes[i]}`;
+}
+
+export class LionInputFile extends ScopedElementsMixin(LocalizeMixin(LionField)) {
+  static get scopedElements() {
+    return {
+      ...super.scopedElements,
+      'lion-uploaded-file-list': LionUploadedFileList,
+    };
+  }
+
+  static get properties() {
+    return {
+      ...super.properties,
+      _uploadedFilesMetaData: {
+        type: Array,
+      },
+      allowedFileExtensions: {
+        type: String,
+        attribute: 'allowed-file-extensions',
+      },
+      allowedFileTypes: {
+        type: String,
+        attribute: 'allowed-file-types',
+      },
+      enableDragAndDrop: {
```

Comment:
So basically (since we use native file input under the hood) drag and drop is enabled, but this flag shows a bigger "drop zone"?
So maybe call it "enable-drop-zone"?